### PR TITLE
updated 99 for in progress

### PR DESCRIPTION
**Pull request by chadyj**
Tweaks for in progress.

### CODE REVIEW

File: ideas/99-confidence.md [hunk 1]
```diff
@@ -90,30 +90,38 @@ Request messages history after background: https://github.com/status-im/status-r
...
+- [#3785 Remove "seen by everyone" from public chat](https://github.com/status-im/status-react/issues/3785)
 
-Testing Days required: <!-- Days required at the end of development for testing -->
+### Iteration 2
+
+After iteration 1 is complete the swarm will meet and discuss the results of the UXR survey, send/receive ratio results, review current UX and discuss future iterations. Work will then be planned for a future iteration, or the swarm will be closed.
+
+## Exit Criteria
+
+- >99% message deliverability from [#3792](https://github.com/status-im/status-react/issues/3792)
```
Comment (lukaszfryc) [inline ideas/99-confidence.md]: Maybe we should also add max time we want messages to be delivered? For example: "95% messages delivered within 5s for a real-time chat". https://github.com/status-im/status-react/issues/3792 could probably measure it

Comment (chadyj) [reply ideas/99-confidence.md]: There was a discussion about the timeout, so that could be our cut off?

Comment (cammellos) [reply ideas/99-confidence.md]: @lukaszfryc  unfortunately is not under our control, as that includes network delays. We can add though the time it takes to dispatch the message and how long it takes after receiving a message to display it to the user.

**Comment by cammellos** [comment]
In general I feel we should go for more granular swarms (or have sub-swarms), as it easier to have more focused initiative, what do you think?

**Comment by chadyj** [comment]
> Are we removing the sample size in trust question because we're confident that enough people will respond? Fine if we are but just want to make sure we have an idea of how many people need to respond to trust results.

Sorry @jpbowen removed that because I wasn't sure how up-to-date it was and assumed you would come up with the right number of participants based on the goal. I can put it back in if the number is solid?

**Comment by jpbowen** [comment]
Let's start with a min of 100 responses to be able to start to compare
level differences - i.e. "somewhat confident" vs. "very confident". This is
assuming something like 10k population size (for beta or mainnet) at 95%
confidence and +/- error of 10%. If you we would like more precision (+/-
5%) then we'll need ~ 4x the number of responses.

Here is a reference:
https://www.surveymonkey.com/mp/sample-size-calculator/?ut_source=help_center

**Comment by chadyj** [comment]
> Let's start with a min of 100 responses @jpbowen 

Done.

**Comment by oskarth** [comment]
@chadyj @rasom Want to resolve conflicts and go ahead and merge this?

**Comment by chadyj** [comment]
> In general I feel we should go for more granular swarms (or have sub-swarms), as it easier to have more focused initiative, what do you think?

Agree. This swarm is rather broad, but part of its work is to identify problems that can be focused on in future iterations or even other swarms. Main work in the first iteration is to get a UXR and delivery reliability benchmark to gauge the issue.

**Pull request merged**